clickstream/src/main/kotlin/clickstream/extension/CSMessageExt.kt

```diff
@@ -73,4 +76,53 @@ public fun MessageLite.eventName(): String? {
         declaredField.isAccessible = true
         declaredField.get(this) as String
     }.getOrNull()
-}
\ No newline at end of file
+}
+
+/**
+ * Converts [MessageLite] to [JSONObject] using reflection.
+ * */
+public fun MessageLite.toJson(): JSONObject {
```

Comment:
do we have test for this function?

Reply:
Added [CSMessageExtTest.kt](https://github.com/gojek/clickstream-android/pull/14/files#diff-b0d592abf898968fa2d7822fa50a3040a4ca1de79d676fc38b20a2887727f000)